.github/workflows/3x.yml

```diff
@@ -18,14 +18,14 @@ jobs:
     name: Checkout & build Phar
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Save repo content as artifact
         uses: actions/upload-artifact@v2
         with:
           name: full-workspace
           path: ${{ github.workspace }}
       - name: Full Composer Install
-        run: composer install
+        run: composer install --dev
```

Comment:
My only concern is that this may increase final .phar file size. Could you please check against https://github.com/pantheon-systems/terminus/releases/tag/3.1.2 and confirm that's not an issue? Please